app/layouts/ProjectLayoutBase.tsx

```diff
@@ -0,0 +1,123 @@
+   */
+  overrideContentPane?: ReactElement
+}
+
+const projectView = ({ project }: PP.Project) => apiq('projectView', { path: { project } })
+
+export async function projectLayoutLoader({ params }: LoaderFunctionArgs) {
+  const { project } = getProjectSelector(params)
+  await queryClient.prefetchQuery(projectView({ project }))
+  return null
+}
+
+export function ProjectLayoutBase({ overrideContentPane }: ProjectLayoutProps) {
+  const navigate = useNavigate()
+  // project will always be there, instance may not
+  const projectSelector = useProjectSelector()
+  const { data: project } = usePrefetchedQuery(projectView(projectSelector))
+
+  const { pathname } = useLocation()
+  useQuickActions(
+    useMemo(
+      () =>
+        [
+          { value: 'Instances', path: pb.instances(projectSelector) },
+          { value: 'Disks', path: pb.disks(projectSelector) },
+          { value: 'Snapshots', path: pb.snapshots(projectSelector) },
+          { value: 'Images', path: pb.projectImages(projectSelector) },
+          { value: 'VPCs', path: pb.vpcs(projectSelector) },
+          { value: 'Floating IPs', path: pb.floatingIps(projectSelector) },
+          { value: 'Access', path: pb.projectAccess(projectSelector) },
+        ]
+          // filter out the entry for the path we're currently on
+          .filter((i) => i.path !== pathname)
+          .map((i) => ({
+            navGroup: `Project '${project.name}'`,
+            value: i.value,
+            onSelect: () => navigate(i.path),
+          })),
+      [pathname, navigate, project.name, projectSelector]
+    )
+  )
+
+  return (
+    <PageContainer>
+      <TopBar systemOrSilo="silo" />
+      <Sidebar>
+        <Sidebar.Nav>
+          <NavLinkItem to={pb.projects()} end>
+            <Folder16Icon />
+            Projects
+          </NavLinkItem>
+          <DocsLinkItem />
+        </Sidebar.Nav>
+        <Divider />
+        <Sidebar.Nav heading={project.name}>
+          <NavLinkItem to={pb.instances(projectSelector)}>
+            <Instances16Icon /> Instances
+          </NavLinkItem>
+          <NavLinkItem to={pb.disks(projectSelector)}>
+            <Storage16Icon /> Disks
+          </NavLinkItem>
+          <NavLinkItem to={pb.snapshots(projectSelector)}>
+            <Snapshots16Icon /> Snapshots
+          </NavLinkItem>
+          <NavLinkItem to={pb.projectImages(projectSelector)}>
+            <Images16Icon title="images" /> Images
+          </NavLinkItem>
+          <NavLinkItem to={pb.vpcs(projectSelector)}>
+            <Networking16Icon /> VPCs
+          </NavLinkItem>
+          <NavLinkItem to={pb.floatingIps(projectSelector)}>
+            <IpGlobal16Icon /> Floating IPs
+          </NavLinkItem>
+          <NavLinkItem to={pb.projectAccess(projectSelector)}>
+            <Access16Icon title="Access" /> Access
+          </NavLinkItem>
+        </Sidebar.Nav>
+      </Sidebar>
+      {overrideContentPane || <ContentPane />}
+    </PageContainer>
+  )
+}
```

Comment:
This is just `ProjectLayout` moved in its entirety. The diff doesn’t show as a rename because the original file is still there, it’s just very short now.